data.sql

```diff
@@ -33,4 +34,90 @@ INSERT INTO animals(
     neutered,
     escape_attempts
   )
-VALUES(4, 'Devimon', '2017-05-12', 11, TRUE, 5);
\ No newline at end of file
+VALUES(4, 'Devimon', '2017-05-12', 11, TRUE, 5);
+INSERT INTO animals (
+    id,
+    name,
+    date_of_birth,
+    escape_attempts,
+    neutered,
+    weight_kg,
+    species
+  )
+VALUES(
+    5,
+    'Charmander',
+    '2020-02-08',
+    0,
+    FALSE,
+    11,
+    null
+  );
+INSERT INTO animals (
+    id,
+    name,
+    date_of_birth,
+    escape_attempts,
+    neutered,
+    weight_kg,
+    species
+  )
+VALUES(6, 'Plantmon', '2021-11-15', 2, TRUE, 5.7, null);
+INSERT INTO animals (
+    id,
+    name,
+    date_of_birth,
+    escape_attempts,
+    neutered,
+    weight_kg,
+    species
+  )
+VALUES(
+    7,
+    'Squirtle',
+    '1993-04-02',
+    3,
+    FALSE,
+    12.13,
+    null
+  );
```

Comment:
- [ ] As specified in the [requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md), the first four values of weight are negative, please kindly fix that.